server/routes/players.js

```diff
@@ -0,0 +1,43 @@
+let Players = require("../models/players");
+
+module.exports = (app) => {
+    // Add new player
+    app.post('/players', (req, res) => {
+        Players.addPlayer(req.body.name, req.body.nickname, req.body.chant).then(
+            (message) => { res.send(message); },
+            (err) => { console.error(err); }
+        );
+    });
+
+    // Fetch all players
+    app.get('/players', (req, res) => {
+        Players.fetchAll().then(
+            (players) => { res.send({players: players}); },
+            (err) => { console.error(err); }  
+        );
+    });
+
+    // Fetch single player
+    app.get('/player/:id', (req, res) => {
+        Players.fetchOne(req.params.id).then(
+            (player) => { res.send(player); },
+            (err) => { console.error(err); } 
+        );
+    });
+
+    // Update a player
+    app.put('/players/:id', (req, res) => {
+        Players.updateOne(req.params.id, req.body.name, req.body.nickname, req.body.chant).then(
```

Comment:
This got merged but this was broken still.  lol

Reply:
Yeah, I realized that immediately afterwards lol.

Reply:
I'm making a new branch, no worries.  I should have fixed it in both at the samt ime.

Reply:
I'll fix it